results/demo/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func main() {
+	kubeconfig := flag.String("kubeconfig", "/root/.kube/config", "absolute path to the kubeconfig file")
+	flag.Parse()
+	config, err := clientcmd.BuildConfigFromFlags("", *kubeconfig)
+	if err != nil {
+		panic(err.Error())
+	}
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
+	}
+	
+	
+			pods, err := clientset.CoreV1().Pods("").List(v1.ListOptions{})
+			if err != nil {
+				panic(err.Error())
+			}
+			for _, pod := range pods.Items {
+				fmt.Printf("%s %s\n", pod.GetName(), pod.GetCreationTimestamp())
+			}
```

Comment:
Run this through gofmt.

Reply:
oh, later I found this one is useless I just used it for debugging watcher. Will delete it.

Reply:
Removed the file since it is unused now.

Reply:
done